### PR TITLE
Folder added : Assignment 1 in sourabh_kanwade/bootstrap/

### PR DESCRIPTION
**Pull request by sourabh-kanwade**

### CODE REVIEW

**Review by Prashant698 — APPROVED** [review]

**Review by somesh6797 — APPROVED** [review]

File: Sourabh_Kanwade/bootstrap/Assignment 1/person_black.svg [hunk 1]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" height="24px" viewBox="0 0 24 24" width="24px" fill="#000000"><path d="M0 0h24v24H0V0z" fill="none"/><path d="M12 12c2.21 0 4-1.79 4-4s-1.79-4-4-4-4 1.79-4 4 1.79 4 4 4zm0 2c-2.67 0-8 1.34-8 4v2h16v-2c0-2.66-5.33-4-8-4z"/></svg>
```
Comment (Pawan1005) [inline Sourabh_Kanwade/bootstrap/Assignment 1/person_black.svg]: add newline

File: Sourabh_Kanwade/bootstrap/Assignment 1/more.svg [hunk 2]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" height="24px" viewBox="0 0 24 24" width="24px" fill="#000000"><path d="M0 0h24v24H0V0z" fill="none"/><path d="M12 8c1.1 0 2-.9 2-2s-.9-2-2-2-2 .9-2 2 .9 2 2 2zm0 2c-1.1 0-2 .9-2 2s.9 2 2 2 2-.9 2-2-.9-2-2-2zm0 6c-1.1 0-2 .9-2 2s.9 2 2 2 2-.9 2-2-.9-2-2-2z"/></svg>
```
Comment (Pawan1005) [inline Sourabh_Kanwade/bootstrap/Assignment 1/more.svg]: add newline

File: Sourabh_Kanwade/bootstrap/Assignment 1/help_outline.svg [hunk 3]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" height="24px" viewBox="0 0 24 24" width="24px" fill="#000000"><path d="M0 0h24v24H0V0z" fill="none"/><path d="M11 18h2v-2h-2v2zm1-16C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 18c-4.41 0-8-3.59-8-8s3.59-8 8-8 8 3.59 8 8-3.59 8-8 8zm0-14c-2.21 0-4 1.79-4 4h2c0-1.1.9-2 2-2s2 .9 2 2c0 2-3 1.75-3 5h2c0-2.25 3-2.5 3-5 0-2.21-1.79-4-4-4z"/></svg>
```
Comment (Pawan1005) [inline Sourabh_Kanwade/bootstrap/Assignment 1/help_outline.svg]: add newline

File: Sourabh_Kanwade/bootstrap/Assignment 1/description.svg [hunk 4]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" height="24px" viewBox="0 0 24 24" width="24px" fill="#ffffff"><path d="M0 0h24v24H0V0z" fill="none"/><path d="M8 16h8v2H8zm0-4h8v2H8zm6-10H6c-1.1 0-2 .9-2 2v16c0 1.1.89 2 1.99 2H18c1.1 0 2-.9 2-2V8l-6-6zm4 18H6V4h7v5h5v11z"/></svg>
```
Comment (Pawan1005) [inline Sourabh_Kanwade/bootstrap/Assignment 1/description.svg]: add newline

**Review by Pawan1005 — CHANGES_REQUESTED** [review]
add newline in svg files

**Review by Pawan1005 — APPROVED** [review]

**Pull request merged**